src/libs/ReportUtils.ts

```diff
@@ -2487,16 +2241,16 @@ function updateReportPreview(iouReport, reportPreviewAction, isPayRequest = fals
                 type: CONST.REPORT.MESSAGE.TYPE.COMMENT,
             },
         ],
-        childLastMoneyRequestComment: comment || reportPreviewAction.childLastMoneyRequestComment,
-        childMoneyRequestCount: reportPreviewAction.childMoneyRequestCount + (isPayRequest ? 0 : 1),
-        childLastReceiptTransactionIDs: hasReceipt ? [transaction.transactionID, ...previousTransactionIDs].join(',') : lastReceiptTransactionIDs,
+        childLastMoneyRequestComment: comment || reportPreviewAction?.childLastMoneyRequestComment,
+        childMoneyRequestCount: (reportPreviewAction?.childMoneyRequestCount ?? 0) + (isPayRequest ? 0 : 1),
+        childLastReceiptTransactionIDs: hasReceipt ? [transaction?.transactionID, ...previousTransactionIDs].join(',') : lastReceiptTransactionIDs,
```

Comment:
@kubabutkiewicz This way `undefined` can be in the array of `childLastReceiptTransactionIDs`